suite-native/module-trading/src/hooks/general/useSectionList.tsx

```diff
@@ -25,21 +27,52 @@ export type SectionHeaderRenderConfig<U> = {
 };
 
 export type ListInternalItemShape<T, U> =
-    // [type, text, key, sectionData]
-    | ['sectionHeader', ReactNode, string, U]
-    // [type, data, config]
-    | ['item', T, ItemRenderConfig<U>];
+    | {
+          type: 'sectionHeader';
+          text: ReactNode;
```

Comment:
Maybe title when we allow it to be a component?
```suggestion
          title: ReactNode;
```

Reply:
Updated in e1240063bb88eb9cdbdc49aaf9607458d3d0a3dd